.changeset/fresh-apricots-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@octopost/core': patch
+---
+
+In this Changeset, in general, were add features regarding a Adjusting Main Content for Mobile Version.
```

Comment:
Não precisa colocar "In this changeset, in general". Pode ser só "were add features regarding a Adjusting Main Content for Mobile Version."

Reply:
isso ai foi o própio changeset que gerou